pkg/util/kube.go

```diff
@@ -667,3 +667,7 @@ func DeleteMultiClusterSecret(namespace string, remoteKubeConfig string, localKu
 	}
 	return err
 }
+
+func CheckDeploymentIsReady(namespace, name string, timeout time.Duration) (string, error) {
```

Comment:
@jwendell could you move this function from kube.go to ocp.go ?
There is an existing func CheckDeploymentsReady in line 564 kube.go from earlier upstream util. We are not using  CheckDeploymentsReady. And thanks for adding a new oc util function.

Reply:
done